extensions/amp-analytics/0.1/test/test-video-analytics.js

```diff
@@ -199,9 +194,79 @@ describes.realWin(
         expect(tracker.sessionObservable_.handlers_.length).to.equal(1);
       });
 
+      it('fires on class selectors', async () => {
```

Comment:
Hmm.. we don't want to change the existing API contract. Currently, for a non-array selector, if the selector is a class name, we will only find the first element that satisfies the selector. 

Can we find a way to keep this behavior? 
i.e. non-array selector only get us 1 element, but array selector can get us multiple elements per selector.